recaptcha/C001056.yaml

```diff
@@ -2,11 +2,9 @@ bioguide: C001056
 contact_form:
   method: POST
   steps:
-    - visit: "http://www.cornyn.senate.gov/contact"
+    - visit: "https://www.cornyn.senate.gov/node/5853"
```

Comment:
why not delete this file now?

Reply:
oops, I thought I had already deleted this file. Just did